platforms/firmata/README.md

```diff
@@ -48,6 +48,63 @@ func main() {
 	gbot.Start()
 }
 ```
+
+## How to Connect
+
+### Upload the Firmata Firmware to the Arduino
+
+This section assumes you're using an Arduino Uno or another compatible board, and a UNIX operating system (OS X or Linux). If you already have the Firmata sketch installed, you can skip straight to the examples.
+
+### OS X
+
+First plug the Arduino into your computer via the USB/serial port.
+A dialog box will appear telling you that a new network interface has been detected.
+Click "Network Preferences...", and when it opens, simply click "Apply".
+
+Once plugged in, use [Gort](http://gort.io)'s `gort scan serial` command to find out your connection info and serial port address:
+
+```
+$ gort scan serial
+```
```

Comment:
these can just be indented 4 spaces, rather than surrounding as a fenced code block